packages/indexer-cli/src/commands/indexer/actions/approve.ts

```diff
@@ -4,7 +4,8 @@ import chalk from 'chalk'
 import { loadValidatedConfig } from '../../../config'
 import { createIndexerManagementClient } from '../../../client'
 import { fixParameters, printObjectOrArray } from '../../../command-helpers'
-import { approveActions } from '../../../actions'
+import { approveActions, fetchActions } from '../../../actions'
+import { ActionStatus } from '@graphprotocol/indexer-common'
 
 const HELP = `
 ${chalk.bold('graph indexer actions approve')} [options] [<actionID1> ...]
```

Comment:
Can you include the new `queued` variant that can be supplied as an argument to this command?

In other commands we've done this by adding another line to the help with the new command format. Using that paradigm we could add another line like:

```
${chalk.bold('graph indexer actions approve')} [options] queued
```